go/lang/best-practice/channel-guarded-with-mutex.yaml

```diff
@@ -16,3 +16,7 @@ rules:
     for more information.
   languages: [go]
   severity: WARNING
+  metadata:
+    category: best-practice
+    technology:
+    - otto
```

Comment:
Same as above, for go libraries would we use their full "path" like `robertkrimen/otto`?